HaskellNet-SSL.cabal

```diff
@@ -37,7 +37,7 @@ library
   other-modules:       Network.HaskellNet.SSL.Internal
   if flag(NoUpperBounds)
     build-depends:     base >= 4,
-                       HaskellNet >= 0.3,
+                       HaskellNet >= 0.3 && < 0.7,
```

Comment:
This is the no upper bounds section! Add the upper bound to the else clause, and remove this one. Then it should be good to go :joy: